server_release_template/web_gui/js/app/setupWizard.js

```diff
@@ -7,7 +7,39 @@ define([
 ], function (_, driverList, wizardTemplate, i18n) {
     return function (alvrSettings) {
 
+        class GPU {
+            constructor() {
+                const gl = document.createElement('canvas').getContext('webgl');
+                const debugInfo = gl.getExtension("WEBGL_debug_renderer_info");
+                var rawGPUInfo = gl.getParameter(debugInfo.UNMASKED_RENDERER_WEBGL);
+
+                this.fullName = rawGPUInfo.match(/((NVIDIA|AMD|Intel)[^\d]*[^\s]+)/)[0];
+                [this.dev, this.name] = this.fullName.split(/(?<=^\S+)\s/);
+            }
+        }
+
+        function CheckGPUSupport(name) {
```

Comment:
Single letter difference between function names is bad, you can put this into CheckGPUSSupport()